delivery_deliverea/models/delivery_carrier.py

```diff
@@ -0,0 +1,538 @@
+            ]
+        )
+        return default_field.id
+
+    delivery_type = fields.Selection(
+        selection_add=[("deliverea", "Deliverea")],
+        ondelete={"deliverea": "set default"},
+    )
+    deliverea_username = fields.Char()
+    deliverea_password = fields.Char()
+    deliverea_distribution_center_id = fields.Many2one(
+        comodel_name="deliverea.distribution.center",
+        string="Deliverea Distribution Center",
+    )
+    deliverea_url_prod = fields.Char(string="Deliverea production environment URL")
+    deliverea_url_test = fields.Char(string="Deliverea test environment URL")
+    deliverea_url_tracking = fields.Char(string="Deliverea tracking URL")
+    deliverea_description = fields.Char(
+        string="Deliverea Articles Description",
+        help="Description of the expedition articles (required for crossborder"
+        " international expeditions if no description by parcel[items] is present)",
+    )
+    deliverea_note_selection_id = fields.Many2one(
+        comodel_name="ir.model.fields",
+        string="Select Note Field",
+        domain="[('model_id', '=', 'stock.picking')]",
+        default=_get_default_field,
+        help="It allows to choose the field that we want "
+        "to show within the carrier observations",
+    )
+    deliverea_default_packaging_id = fields.Many2one(
+        comodel_name="stock.package.type",
+        string="Default Packaging Type",
+        domain=[("package_carrier_type", "=", "deliverea")],
+        help="Default weight, height, width and length for packages",
+    )
+    deliverea_notifications_sms = fields.Boolean(
+        string="Notify by sms",
+        help="The carrier will send info sms(s) to the final customer",
+    )
+    deliverea_notifications_email = fields.Boolean(
+        string="Notify by email",
+        help="The carrier will send info email(s) to the final customer",
+    )
+    deliverea_saturday_delivery = fields.Boolean(
+        string="Saturday Delivery",
+        help="Whether or not the expedition should be delivered on Saturday"
+        " or wait until next Monday (availability depends on carrier and service)",
+    )
+    deliverea_return_label = fields.Boolean(
+        string="Return Label",
+        help="Whether or not to include dormant return label in case the final customer"
+        " wants to return the expedition (availability depends on carrier and service)",
+    )
+    deliverea_return_proof_delivery = fields.Boolean(
+        string="Return Proof Delivery",
+        help="Whether or not to a proof of delivery should be returned"
+        " back to origin (availability depends on carrier and service)",
+    )
+    deliverea_hide_sender = fields.Boolean(
+        string="Hide Sender",
+        help="Whether or not to hide sender information in the printed label"
+        " (availability depends on carrier and service)",
+    )
+    deliverea_carrier_service = fields.Many2one(
```

Comment:
All m2o should end with `_id` by convention.

Reply:
Changed the field name.